examples/example.py

```diff
@@ -8,24 +8,23 @@
    >>> import json, pprint, pyarrow.parquet as pq
    >>> pprint.pprint(json.loads(pq.read_schema("example.parquet").metadata[b"geo"]))
    {'columns': {'geometry': {'bbox': [-180.0, -90.0, 180.0, 83.6451],
-                             'crs': 'GEOGCRS["WGS 84",ENSEMBLE["World Geodetic '
-                                    'System 1984 ensemble",MEMBER["World Geodetic '
-                                    'System 1984 (Transit)"],MEMBER["World '
-                                    'Geodetic System 1984 (G730)"],MEMBER["World '
-                                    'Geodetic System 1984 (G873)"],MEMBER["World '
-                                    'Geodetic System 1984 (G1150)"],MEMBER["World '
-                                    'Geodetic System 1984 (G1674)"],MEMBER["World '
-                                    'Geodetic System 1984 (G1762)"],MEMBER["World '
+                             'crs': 'GEOGCRS["WGS 84 (CRS84)",ENSEMBLE["World '
```

Comment:
I find that this output is hard to read because lines are very long... inside of a dict... inside of Python output ... inside of a comment... I also personally don't find it very clean to update this code comment in each PR, because the git diffs are hard to parse.

I'd propose that for our examples we have a Parquet file and then a neighboring JSON file containing the metadata. So for example:

```
example1.parquet
// plain text schema that matches what's stored in example1.parquet
example1_schema.json

example2.parquet
// plain text schema that matches what's stored in example2.parquet
example2_schema.json
```

We could have a CI step that verifies the plain text schema in `*_schema.json` matches what's actually stored in `example1.parquet`.

I think this setup would make it easier for people browsing the spec to see examples of what the schema looks like without needing to open up a file.

Reply:
+1 - I like the idea of a neighboring example file that show's what's stored in it.

And yeah, seems in line with the general idea to make JSON schema a part of the core repo. I haven't made an issue for that one either. Feel free to make that issue, and include yours or make a linked issue.

Reply:
good idea @kylebarron!

Reply:
I made #65 to discuss this.